R/import.R

```diff
@@ -163,6 +163,43 @@ import.clipboard <- function(header = TRUE, sep = "\t", ...) {
     }
 }
 
+import.sql <- function(){
+    switch(db_type,
+           mysql = import.mysql(),
+           postgresql = import.pg(),
+           sqlite = import.sqlite(),
+           stop("Unrecognized file format"))
+}
+
+import.mysql <- function(dbname, host, port, username, password, table, ...){
+  if (!requireNamespace("dplyr", quitely = TRUE)){
```

Comment:
We don't need these `requireNamespace` calls. The package is going to import dplyr, so that will always be true.